handlers_test.go

```diff
@@ -62,7 +62,7 @@ func TestTokenAuthorization(t *testing.T) {
 		newData, _ := url.ParseQuery(data.Encode())
 		newData.Del(key)
 		rr := testResponse(t, tokenEndpoint, m.Token, http.MethodPost, newData)
-		assert.Equalf(t, rr.Code, 400, "Should be 400 but was %d, even though %s is missing", rr.Code, key)
+		assert.GreaterOrEqualf(t, rr.Code, 400, "Should be error but was %d, even though %s is missing", rr.Code, key)
```

Comment:
lets do exact error code tests and use the `http.Status...` constants instead of raw ints